Sources_v3/Controllers/ChannelController/ChannelController.swift

```diff
@@ -284,14 +291,15 @@ public class ChannelControllerGeneric<ExtraData: ExtraDataTypes>: Controller, De
         do {
             try channelObserver.startObserving()
             try messagesObserver.startObserving()
+            completion?(nil)
```

Comment:
Why we even use the completion here? Why not just throw an error? We already talked about it here https://github.com/GetStream/stream-chat-swift/pull/406#discussion_r472901467 and it seems to me, that with the recent changes, it makes much more sense. It will simplify the double-completion brain breaker in `startUpdating()`

Reply:
Yes, definitely. I've thought about this after opening the PR. It doesn't need to have a completion, just a return value, or throws would be enough